src/NUnitFramework/framework/Attributes/RetryAttribute.cs

```diff
@@ -36,18 +29,18 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// RepeatAttribute may be applied to test case in order
-    /// to run it multiple times.
+    /// <see cref="RetryAttribute" /> is used on a test method to specify that it should
+    /// be rerun if it fails, up to a maximum number of times.
     /// </summary>
     [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = false)]
     public class RetryAttribute : PropertyAttribute, IWrapSetUpTearDown
     {
         private int _count;
 
         /// <summary>
-        /// Construct a RepeatAttribute
+        /// Construct a <see cref="RetryAttribute" />
         /// </summary>
-        /// <param name="count">The number of times to run the test</param>
+        /// <param name="count">The maximum number of times the test should be rerun if it fails</param>
```

Comment:
Oh. This should say <code>The maximum number of times the test should be **run**</code>, not `rerun`.